ld_script.txt

```diff
@@ -193,6 +193,7 @@ SECTIONS {
         asm/pokedex_cry_screen.o(.text);
         src/coins.o(.text);
         src/landmark.o(.text);
+		src/fldeff_strength.o(.text);
```

Comment:
tabs vs spaces

Reply:
It was a dirty commit to just claim the file. But now the file has been fully decomped with all of these points already kept in mind.